src/app/services/auth.service.ts

```diff
@@ -7,24 +7,27 @@ import { HttpClient } from '@angular/common/http';
 })
 export class AuthService {
   wrongData: boolean = false;
+  signSuccess: boolean = false;
 
   get userIsLogged(): boolean {
-    const data: any = localStorage.getItem('userIsLogged');
-    const userIsLogged: boolean = JSON.parse(data);
-    return data ? userIsLogged : false;
+    const localStorageData: any = localStorage.getItem('userIsLogged');
+    const userIsLogged: boolean = JSON.parse(localStorageData);
+    return localStorageData ? userIsLogged : false;
   }
 
   constructor(private router: Router, private http: HttpClient) {}
 
   userLogin(name: any, pass: any): void {
     this.http.get<any>('http://localhost:3000/signupUsersList').subscribe({
       next: (res) => {
-        const user: boolean = res.find((db: any) => {
-          return db.username === name && db.password === pass;
+        const userLogged: boolean = res.find((db: any) => {
+          return (
+            (db.username === name || db.email === name) && db.password === pass
+          );
         });
-        if (user) {
-          localStorage.setItem('userIsLogged', JSON.stringify(true));
 
+        if (userLogged) {
+          localStorage.setItem('userIsLogged', JSON.stringify(true));
```

Comment:
Instead of 'JSON.stringify(true)' just try true

Reply:
Argument of type 'boolean' is not assignable to parameter of type 'string'